workshop/views.py

```diff
@@ -116,7 +116,7 @@ class CouncilDetailView(generics.RetrieveUpdateAPIView):
     """
     # pylint: disable=no-member
     queryset = Council.objects.all()
-    permission_classes = (permissions.AllowAny,)
+    permission_classes = (AllowParticularCouncilHead)
```

Comment:
```suggestion
    permission_classes = (AllowParticularCouncilHead,)
```
A tuple with a single element should have a comma at the end, otherwise it will give `500 Error` on accessing the endpoint.